backend/lib/kubernetes-client/util.js

```diff
@@ -0,0 +1,293 @@
+//
+// Copyright (c) 2019 by SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const { HTTPError } = require('got')
+const logger = require('../logger')
+const { GatewayTimeout, InternalServerError } = require('../errors')
+
+function ctors (filename) {
+  const basename = path.basename(filename)
+  const dirname = path.dirname(filename)
+  const baseDir = path.relative(__dirname, dirname)
+  const files = fs.readdirSync(dirname)
+  const data = {}
+  for (const file of files) {
+    if (file !== basename) {
+      const Ctor = require(`./${baseDir}/${file}`)
+      const { name, names: { plural } = {} } = Ctor
+      data[plural || name.toLowerCase()] = Ctor
```

Comment:
change to if/else and add comment for name.toLowerCase() case

Reply:
✔️